spec/features/application_spec.rb

```diff
@@ -9,13 +9,6 @@
   end
 
   describe 'login link' do
```

Comment:
'link' could be removed to prevent confusion in the future, as the relevant test is now gone